.github/workflows/ci.yml

```diff
@@ -198,7 +198,7 @@ jobs:
       - name: Install system dependencies
         run: |
           sudo apt update
-          sudo apt install libgirepository1.0-dev python3-testresources
+          sudo apt install libgirepository1.0-dev libayatana-appindicator3-1 gir1.2-ayatanaappindicator3-0.1 python3-testresources
```

Comment:
Same as above. Also, please note that if you intend to move to Ayatana's AppIndicator implementation, you'll have to also update the AppImage build recipe (`appimage/AppImageBuilder.yml`) to include all dependencies required by Aytana.

Reply:
From what it looks like does it not required more dependencies than what's already included, tho i'll have a bit deeper look just to be sure

Reply:
Please do. Because I think at the very least you'll have to replace this dependency:

https://github.com/DevilXD/TwitchDropsMiner/blob/80f3e1b5f9002834b935ec3ca4927767bd1f8801/appimage/AppImageBuilder.yml#L51

By `gir1.2-ayatanaappindicator3-0.1`.

Reply:
Yee that i already released, gonna do a push after a bit more research on this one exactly